scripts/linters/custom_eslint_checks/rules/no-multiline-disable.js

```diff
@@ -0,0 +1,64 @@
+// Copyright 2020 The Oppia Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @fileoverview Lint check to ensure that there is a break after parenthesis
+ * in case of multiline hanging indentation.
+ */
+
+'use strict';
+
+module.exports = {
+  meta: {
+    type: 'layout',
+    docs: {
+      description: (
+        'Lint check to ensure that there is a break after parenthesis in case' +
+        ' of multiline hanging indentation'),
+      category: 'Stylistic Issues',
```

Comment:
Not a stylistic issue, right?

Reply:
This is not resolved?

Reply:
I've checked from the eslint codebase, it will come under Stylistic issue.

Reply:
Thanks for the info. Add the link for other reviewers!

Reply:
Link: https://github.com/eslint/eslint/blob/89292084bf91ba5ae5bf966c6c56fa3da139ce57/lib/rules/capitalized-comments.js#L108

Reply:
The rule link you have shared is not similar to what you have implemented here. Should we change the category to "Possible Errors"?